src/clients/api/mutations/useEnterMarkets.ts

```diff
@@ -22,7 +22,19 @@ const useEnterMarkets = (
         comptrollerContract,
         ...params,
       }),
-    options,
+    {
+      ...options,
+      onSuccess: (
+        data: void,
+        variables: Omit<IEnterMarketsInput, 'comptrollerContract'>,
+        context: unknown,
+      ) => {
+        if (options?.onSuccess) {
+          options.onSuccess(data, variables, context);
+        }
+        queryClient.invalidateQueries(FunctionKey.GET_ASSETS_IN_ACCOUNT);
```

Comment:
I'd call `options.onSuccess` after the queries invalidation, since the mutation has only finished executing after the queries have been invalidated.